versions/3.0.4.md

```diff
@@ -96,6 +100,10 @@ Some examples of possible media type definitions:
 The HTTP Status Codes are used to indicate the status of the executed operation. 
 Status codes SHOULD be selected from the available status codes registered in the [IANA Status Code Registry](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).
 
+##### <a name="httpAndCaseSensitivity"></a>HTTP and Case Sensitivity
```

Comment:
```suggestion
##### <a name="httpAndCaseSensitivity"></a>HTTP and Case Sensitivity

```

(some markdown renderers prefer a blank line after titles and most of the document does have them)

Reply:
I'm sure the only reason I didn't put one is that I saw one that didn't have them.  I prefer them set off by blank lines myself.